ocular/tests/single_node_chain_txs.rs

```diff
@@ -566,6 +544,134 @@ fn local_single_node_chain_test() {
 
             // Assert permission error since acct delegation permission was revoked
             assert_eq!(&actual_msg_exec_commit_response.deliver_tx.log.to_string()[..82], "failed to execute message; message index: 0: authorization not found: unauthorized");
+
+            // Expected MsgMultiSend
```

Comment:
Nit: More of a stylistic thing, but the rest of the expected msgs are outside the docker container, I think we should either bring these out or pull the rest in for consistency